.circleci/install-content-server.sh

```diff
@@ -6,7 +6,7 @@ if grep -e "fxa-content-server" -e 'all' $DIR/../packages/test.list; then
   sudo apt-get install -y graphicsmagick
   mkdir -p config
```

Comment:
Should this line be `server/config` too?

Reply:
Good point. Fixing on a branch targetted at train-135, another good point.